tariff-management-service/api/src/main/java/com/microservices/tariffmanagement/api/feign/TariffServiceClient.java

```diff
@@ -0,0 +1,20 @@
+package com.microservices.tariffmanagement.api.feign;
+
+import com.microservices.tariffmanagement.api.dto.TariffDto;
+import com.microservices.tariffmanagement.api.service.TariffService;
+import org.springframework.cloud.openfeign.FeignClient;
+import org.springframework.context.annotation.Primary;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+
+import java.util.List;
+
+@Primary
+@FeignClient(value = "tariff-management-service/tariffs")
+public interface TariffServiceClient extends TariffService {
```

Comment:
Please provide more documentation. This moments are really non trivial and unclear and need to be WELL documented

Reply:
fixed